ir.h

```diff
@@ -338,9 +357,7 @@ typedef union _ir_val {
 			uint32_t                   u32;
 			int32_t                    i32;
 			float                      f;
-#ifndef IR_64
-            uintptr_t                  addr;
-#endif
+			ADDR_MEMBER
```

Comment:
Why do we need this additional ADDR_MEMBER macro?

Reply:
It's MSVC that refuses to compile this piece. In the current version of the code the `#ifdef` is used inside of a `IR_STRUCT_LOHI` macro call, so seems refusing to compile it is sane. What happens there I think is that it would take the preprocessor directives and put them as C without actually interpreting the `#ifdef`, so that would cause a syntax error.

Thanks

Reply:
right. I'm not sure how this works with GCC :)